src/checkout/index.ts

```diff
@@ -0,0 +1,53 @@
+export {
+  useCheckout,
+  CheckoutProvider,
+  CheckoutContextValue,
+} from './components/CheckoutProvider';
+export * from './types';
+import React from 'react';
+import createElementComponent from '../components/createElementComponent';
+import {isServer} from '../utils/isServer';
+import {
+  CurrencySelectorElementComponent,
+  BillingAddressElementComponent,
+  ShippingAddressElementComponent,
+  CheckoutPaymentElementComponent,
+  CheckoutExpressCheckoutElementComponent,
+} from './types';
+
```

Comment:
I realized we should also include the `TaxIdElement` component in this file (and the types file), so developers don't need to import from both the default directory and the /checkout directory to use TaxIdElement!